README.md

```diff
@@ -71,3 +71,24 @@ docker run --env-file .env --add-host=host.docker.internal:host-gateway --detach
 
 And you're done! This mini-tutorial doesn't support the leaderboard, though. You also may wish to set up a simple
 docker compose file. [Join the discord](https://valk.sh/discord) to learn more!
+
+
+# Using Docker Compose
+You can grab the file from [here](<https://github.com/PBOwner/xpd/blob/prod/docker-compose.yml>)
```

Comment:
I think we need to fix the links here. IIRC you can directly link files with `./docker-compose.yml`, which would make it nice and repo-agnostic.

Reply:
this is actually, for this instance, [here](/docker-compose.yml) for relative rather than absolute.